index.php

```diff
@@ -43,7 +43,7 @@ function dataDisplay($dataInput)
     <nav>
         <div>
             <form>
-                <input type="text" placeholder="search books"><input type="submit" value="search">
+                <input type="search" placeholder="search books">
```

Comment:
Little bit of scope creep here, as this isn't part of story 2! You can leave it in, though.